source/dlangbot/github.d

```diff
@@ -199,8 +200,17 @@ Json[] tryMerge(in ref PullRequest pr, MergeMethod method)
         return commits;
     }
 
+    auto labelName = method.labelName;
+    auto events = ghGetRequest(pr.eventsURL).readJson[]
+        .retro
+        .filter!(e => e["event"] == "labeled" && e["label"]["name"] == labelName);
```

Comment:
FWIW the API seems to be rather restrictive:

> Events support pagination, however the per_page option is unsupported. The fixed page size is 30 items. Fetching up to ten pages is supported, for a total of 300 events.

> Only events created within the past 90 days will be included in timelines. Events older than 90 days will not be included (even if the total number of events in the timeline is less than 300).

I hope this won't be a huge issue in practice